usr/src/test/zfs-tests/tests/functional/cli_root/zfs_receive/zfs_receive_010_pos.ksh

```diff
@@ -137,6 +137,11 @@ $TOUCH $mntpnt2/f18
 $RM $mntpnt/h17
 $RM $mntpnt2/h*
 
+# Add empty objects to $fs to exercise dmu_traverse code
+for i in `seq 1 100`; do
```

Comment:
Some distros do for virtual methadone to Linux users.  OmniOS has it in /bin and in /usr/gnu/bin.  If a native illumos seq(1) ever happened, the /bin/ would be replaced in favor of its native version.